changelogs/fragments/419-sflow-max-header-size.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - sonic_ldap - Add max header size support in sonic_sflow module(https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/419)
```

Comment:
Please add a space between the word "module" and the "(" character.
Add a period after the ")" character as well.
The fix should look like this: 
- sonic_ldap - Add max header size support in sonic_sflow module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/419).